apps/src/componentLibrary/dropdown/simpleDropdown/SimpleDropdown.tsx

```diff
@@ -92,6 +104,9 @@ const SimpleDropdown: React.FunctionComponent<SimpleDropdownProps> = ({
           value={selectedValue}
           id={id}
           disabled={disabled}
+          className={classNames({
+            [moduleStyles.error]: errorMessage,
```

Comment:
Let's please rename this className to .hasError so that it'll be more sematic and easier to understand